README.md

```diff
@@ -67,33 +73,29 @@ If for some reason, you are not able to access your phone, you can use the adb (
 4. Open a terminal and type `adb devices`. This should show your phone. If it does, type `adb install [path-to-apk]`.
 
 ## Manual installation
-This section, is if you don't like appimages, or want to run the game natively. Unless you have a very good reason for it, please use the AppImage instead!
+This section, is if you don't like AppImages, or want to run the game natively. Unless you have a very good reason for it, please use the AppImage instead!
+If you cannot run AM2R after installing the packages, use `ldd` in order to find out which packages are missing.
 
 ### Dependencies
 As said above, these dependencies are **only** needed if you want to run the game natively.
 
 ### Arch (including Manjaro, EndeavourOS, RebornOS, etc.)
 Make sure that multilib is enabled, as this is a 32-bit application and Arch does not do so by default.
 To enable it, go to `/etc/pacman.conf`, search for `[multilib]`, and make sure that both this and the next line are uncommented. After that, install the following packages:  
-`sudo pacman -S --needed unzip sed xdelta3 lib32-openal lib32-openssl-1.0 lib32-libcurl-compat lib32-libpulse lib32-gcc-libs lib32-libxxf86vm lib32-libglvnd lib32-libxrandr lib32-glu`
+`sudo pacman -S --needed unzip patchelf sed xdelta3 lib32-openal lib32-libcurl lib32-libpulse lib32-gcc-libs lib32-libxxf86vm lib32-libglvnd lib32-libxrandr lib32-glu`
 
 ### Debian (including Ubuntu, Mint, PopOS, etc.)
-`sudo apt install unzip sed xdelta3 libc6:i386 libstdc++6:i386 zlib1g-dev:i386 libxxf86vm1:i386 libcurl3:i386 libssl1.0:i386 libopenal1:i386 libxrandr2:i386 libglu1:i386`
+Make sure that the i386 architecture is enabled for you, as libopenal1 requires the 32-bit version, and Ubuntu does not enable 32-bit support by default.
```

Comment:
Should be "Make sure you enable the i386 architecture, ..."

Remove comma after 'version'.